### PR TITLE
Check for duplicate incident before adding to database

### PR DESCRIPTION
**Pull request by jduell12**
# Description
Fixes the issue where the checking for duplicates functionality was allowing duplicates through. Added function to add the state abbreviation to each incident per front end request. 

## Type of change
Please delete options that are not relevant.
- Bug fix (non-breaking change which fixes an issue)

## Change Status
- Complete, tested, ready to review and merge

# How Has This Been Tested?
- Jest

### CODE REVIEW

File: api/incidents/incidentsModel.js [hunk 1]
```diff
@@ -14,29 +15,32 @@ async function getAllIncidents() {
...
+    if (tagList[0] != '') {
+      for (let i = 0; i < tagList.length; i++) {
+        await Tags.createTags(tagList[i], incidentID[0]);
+      }
     }
   }
 }
 
-async function showAllIncidents(limit, offset) {
+async function showAllIncidents(limit = 10, offset = 0) {
```
Comment (tonomb) [inline api/incidents/incidentsModel.js]: Doesn't this add a default value to limit and offset?  What happens if you what to get all the data?

**Review by tonomb — CHANGES_REQUESTED** [review]
I just have douts about the defaults for limit and offset, Otherwise looks great!!!

File: api/incidents/incidentsModel.js (hunk 1 above)
Comment (jduell12) [reply api/incidents/incidentsModel.js]: I have to set some default value for limit otherwise if nothing is passed it throws an error. I could also get the length of the current db and use that as a default limit.

Comment (jduell12) [reply api/incidents/incidentsModel.js]: Running some more tests I can't put None or null as the default value. Leaving the default blank while console logging an error doesn't seem to actually break the server tests. I took the default limit out and if needed we can add it back in or create another endpoint that doesn't have a limit or offset.
